etcd/etcd-live-http-to-https-migration.md

```diff
@@ -1,73 +1,73 @@
+If etcd isn't listening on port 4001, it must also be reconfigured. If you used cloud-config to spin up your machines, you can retrieve the `ETCD_LISTEN_CLIENT_URLS` value from `/run/systemd/system/etcd2.service.d/20-cloudinit.conf` to verify the etcd ports:
 
 ```sh
-grep ETCD_LISTEN_CLIENT_URLS /run/systemd/system/etcd2.service.d/20-cloudinit.conf
+$ grep ETCD_LISTEN_CLIENT_URLS /run/systemd/system/etcd2.service.d/20-cloudinit.conf
 Environment="ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379"
 ```
 
-Then create `/etc/systemd/system/etcd2.service.d/` directory if it doesn't exist and create [drop-in][drop-ins] file: `/etc/systemd/system/etcd2.service.d/25-insecure_localhost.conf`. Use value we've retrieved above and append `http://127.0.0.1:4001` URL at the end:
+In this case etcd is listening only on port 2379. We'll add port 4001 with a systemd [drop-in][drop-ins] unit file. Create the file `/etc/systemd/system/etcd2.service.d/25-insecure_localhost.conf`. In this file, write an excerpt that appends the new URL on port 4001 to the existing value we retrieved in the step above:
 
 ```
 [Service]
 Environment="ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379,http://127.0.0.1:4001"
 ```
 
-Then run `systemctl daemon-reload` and `systemctl restart etcd2` to restart etcd. Check cluster availability using [`etcdctl`][etcdctl] commands listed below:
+Run `systemctl daemon-reload` followed by `systemctl restart etcd2` to restart etcd. Check cluster status using the [`etcdctl`][etcdctl] commands:
 
 ```sh
-etcdctl member list
-etcdctl cluster-health
+$ etcdctl member list
+$ etcdctl cluster-health
 ```
 
-Repeat these steps on each CoreOS host which has etcd member.
+Repeat these steps on each etcd cluster member.
 
-### Generate TLS keypairs
+### Generate TLS Key Pairs
 
-Follow [generate self signed certificates][self-signed-ca] doc to generate self-signed certificates.
+Follow the guide to [generating self-signed certificates][self-signed-ca] to create the certificate/key pairs needed for each etcd cluster member.
 
-### Copy Keypairs
+### Copy Key Pairs to Nodes
 
-Let's assume we have three CoreOS hosts with three etcd cluster members: server1, server2, server3 and corresponding IP addresses: 172.16.0.101, 172.16.0.102, 172.16.0.103. We will use following keypairs filenames in our example:
+Assume we have three CoreOS machines, running three etcd cluster members: server1, server2, server3; with corresponding IP addresses: 172.16.0.101, 172.16.0.102, and 172.16.0.103. We will use the following key pair file names in our example:
 
 ```
 server1.pem
 server1-key.pem
 ```
 
-Create `/etc/etcd2` directory, copy there corresponding certificate and key and set file permissions:
+Create the `/etc/etcd2` directory, then copy the corresponding certificate and key there. Set permissions to secure the directory and key file:
```

Comment:
I'd like to have users actually generate the key pairs on the server, rather than shove them around the network, but don't know offhand if this is possible.  If it's not we should investigate adding the necessary tools to the distro.  This may be easy though since apparently OpenSSL and OpenSSH generate mutually-compatible keys.